lib/w_common_browser.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016 Workiva Inc.
```

Comment:
I agree we should move toward importing the individual pieces. Should we deprecate `lib/w_common.dart` and delete this file @evanweible-wf ?